src/main/kotlin/day6/Actor.kt

```
@@ -0,0 +1,3 @@
+package day6
+
+abstract class Actor(open var name: String) {}
```

detekt (1.23.7)

[warning] The class or object Actor is empty.

(detekt.empty-blocks.EmptyClassBlock)